src/tardev-snapshotter/src/snapshotter.rs

```diff
@@ -1008,15 +1013,77 @@ impl TarDevSnapshotter {
                     }
                 }
 
-                trace!("Appending index to {:?}", &base_name);
-                file.rewind().context("failed to rewind the file handle")?;
-                tarindex::append_index(&mut file).context("failed to append tar index")?;
+                // Create an erofs metadata using mkfs.erofs
+                let erofs_path = base_name.with_extension("erofs");
```

Comment:
Technically not accurate, since you later append the merkle tree.

Reply:
will change to layer_path